api/managementpb/dbaas/kubernetes.proto

```diff
@@ -63,6 +63,24 @@ message UnregisterKubernetesClusterRequest {
 
 message UnregisterKubernetesClusterResponse {}
 
+message GetKubernetesClusterRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message GetKubernetesClusterResponse {
+  // Kubernetes auth.
+  KubeAuth kube_auth = 1 [
+    (validator.field) = {
+      msg_exists: true
+    }
```

Comment:
I guess we do not need validator on response

Reply:
Fixed. Thanks.